.vscode/settings.json

```diff
@@ -1,8 +1,3 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib",
-  "editor.tabSize": 2,
```

Comment:
Intentional removal?

Reply:
Yeah, this is covered off by EditorConfig. It was more the "autoformat" rule that I think was causing the a real issue